packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.stories.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { storyWithProps } from '../../storybook-email';
+
+export default {
+  title: 'Layouts/FxA',
```

Comment:
If we're gonna call this "Layouts" and not place it under the Emails section, can we rename the other ones from "Emails" to "Templates"?

Reply:
Huh I see your point. I guess it felt like those are the actual emails that get sent, so the "emails" title made sense and may be more clear from a product stand point... I was going to suggest changing this to "Email Layouts" and leaving it as "Emails," but maybe you're right that "Layouts and "Templates" makes sense. If we ever wanted to show partials in the future, it'd make sense to have the separation as you have it too.